### ISSUE TITLE
Is the input supposed to be rgb or bgr?

### ISSUE THREAD

**Issue by kfeng123** [youngwanLEE/vovnet-detectron2#16]
the default config of detectron2 is bgr. Is it the case here?

**Comment by aishgupta** [comment]
Yes, the models in Detectron2 are trained on the BGR format. The image is converted to RGB format before visualizing by the DefaultPredictor.
